### ISSUE TITLE
train/validation split ratio for hyper-parameter tuning in CIFAR-10?

### ISSUE THREAD

**Issue by alegonz** [google-research/simclr#28]
Hello, thank you for open sourcing this interesting work.

As asked in the title, could you share the train/validation split ratio you used when tuning the hyper-parameters for the CIFAR-10 experiments?

**Comment by chentingpc** [comment]
We didn't do extensive hyper-parameter selection for CIFAR (except searching for a good learning rate), so we simply follow the default train/test split. The effects of other hyper-parameters can be found in Figure B4/B5 in the appendix of the paper.

**Comment by alegonz** [comment]
I see, thank you for your reply!
